src/endpoints/connectToScorebot.ts

```diff
@@ -28,7 +28,9 @@ export const connectToScorebot = (config: HLTVConfig) => ({
       .pop()!
     const matchId = $('#scoreboardElement').attr('data-scorebot-id')
 
-    const socket = io.connect(url)
+    const socket = io.connect(url, {
+      agent: !config.httpAgent
```

Comment:
The documentation for agent in socket.io is very strange:

Option | Default value | Description
--- | --- | ---
agent | false | the http.Agent to use

Are you sure this will work?

Reply:
Yes, I am able to trace it via Wireshark. I've setup http connection to proxy and used https-proxy-agent package. 

![image](https://user-images.githubusercontent.com/9255285/72909403-a218f500-3d3f-11ea-9b5b-44fe6feda55e.png)